securesignals-ima-devapp/src/main/res/drawable-v24/ic_launcher_foreground.xml

```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Android Studio adds *loads* of fancy resources around the app icon. For the Dev Apps though, we don't need them. If you check the other Dev App, we basically just include:
 - ic_launcher.webp (for different densities)
 - ic_laucher_round.web (for different densities)

Let's remove all the fancy xml files please.